benchmark/framework/parser/file_access.go

```diff
@@ -0,0 +1,134 @@
+package bparser
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+	"time"
+)
+
+var (
+	outputDir      = "./output/"
+	sociLogs       = outputDir + "soci-snapshotter-stderr"
+	FileAccessLogs = outputDir + "fileAccessPatterns"
```

Comment:
This does not tell any details about container image. Probably, this information can be embedded somewhere, e.g. output file name, or be the part of fileAccessPatterns file.